src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,281 @@
+package duke.parser;
+import duke.DukeException;
+import duke.command.AddCommand;
+import duke.command.ByeCommand;
+import duke.command.Command;
+import duke.command.DeleteCommand;
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.MarkCommand;
+import duke.task.DeadLine;
+import duke.task.Event;
+import duke.task.ToDo;
+
+import java.util.Arrays;
+import java.util.Calendar;
+import java.util.Date;
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.time.LocalDateTime;
+
+/**
+ * Represents a formatting object that formats inputs in known formats.
+ * Reformats these inputs into different forms depending on the Command.
+ */
+public class Parser {
+	/**
+	 * Returns the integer value of a month.
+	 * @param month String representation of month.
+	 * @return Integer representation of month
+	 * @throws IllegalArgumentException Throws exception should the month not be recognised.
+	 */
+	public static String monthValue(String month) throws IllegalArgumentException {
```

Comment:
Perhaps it would be better to name this function `getMonthValue` instead so that the method name is a verb?